### ISSUE TITLE
Colorize pdb

### ISSUE THREAD

**Issue by gaogaotiantian** [python/cpython#113081]
# Feature or enhancement

### Proposal:

`pdb` is one of the most interactive module in stdlib. Now that we already put ANSI colors in traceback, we should do it in `pdb` as well.

We can start small, but I think eventually we want `pdb` colorized well.

### Has this already been discussed elsewhere?

This is a minor feature, which does not need previous discussion elsewhere

### Linked PRs
* gh-113082
* gh-133355

**Comment by ZeroIntensity** [comment]
Looking through Hugo's dashboard: is this done?

**Comment by gaogaotiantian** [comment]
I don't think so. This is a rather long term issue - the plan is to fully colorize it.
